docs-source/content/samples/simple/azure-kubernetes-service/troubleshooting.md

```diff
@@ -0,0 +1,334 @@
+kube-system                   kubernetes-dashboard-849d5c99ff-hrz2w       1/1     Running   0          3h44m
+kube-system                   metrics-server-7f5b4f6d8c-snnbt             1/1     Running   0          3h44m
+kube-system                   omsagent-8tf4j                              1/1     Running   0          3h43m
+kube-system                   omsagent-n9b7k                              1/1     Running   0          3h42m
+kube-system                   omsagent-rcmgr                              1/1     Running   0          3h43m
+kube-system                   omsagent-rs-787ff54d9d-w7tp5                1/1     Running   0          3h44m
+kube-system                   tunnelfront-794845c84b-v9f98                1/1     Running   0          3h44m
+```
+
+##### WebLogic Kubernetes Operator ErrImagePull
+
+If you got an error of **ErrImagePull** from pod status, use `docker pull` to check the operator image. If an error occurs, you can switch to a version that is greater than `3.1.1`.
+
+```bash
+$ docker pull ghcr.io/oracle/weblogic-kubernetes-operator:<version>
+
+# Example: pull 3.1.1.
+$ docker pull ghcr.io/oracle/weblogic-kubernetes-operator:3.1.1
+3.1.1: Pulling from oracle/weblogic-kubernetes-operator
+980316e41237: Pull complete
+c980371d97ea: Pull complete
+db19c8ff0d12: Pull complete
+550f44317ae5: Pull complete
+be2e701f5ee0: Pull complete
+1cb891615559: Pull complete
+4f4fb700ef54: Pull complete
+Digest: sha256:6b060ec1989fcb26e1acb0d0b906d81fce6b8ec2e0a30fa2b9d9099290eb6416
+Status: Downloaded newer image for ghcr.io/oracle/weblogic-kubernetes-operator:3.1.1
+ghcr.io/oracle/weblogic-kubernetes-operator:3.1.1
+```
+
+#### Cannot attach ACR due to not being Owner of subscription
+
+If you're unable to create an ACR and you're using a Service Principal, you can use manual Role Assignments to grant access to the ACR as described in [Azure Container Registry authentication with service principals](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-service-principal).
+    
+First, find the `objectId` of the Service Principal used when the AKS cluster was created. You will need the output from `az ad sp create-for-rbac`, which you were directed to save to a file.  Within the output, you need the value of the `name` property. It will start with `http`.  Get the `objectId` with this command.
+
+```bash
+$ az ad sp show --id http://<your-name-from-the-saved-output> | grep objectId
+"objectId": "nror4p30-qnoq-4129-o89r-p60n71805npp",
+```
+
+Next, assign the `acrpull` role to that Service Principal with this command.
+
+```bash
+$ az role assignment create --assignee-object-id <your-objectId-from-above> --scope $AKS_PERS_RESOURCE_GROUP --role acrpull
+{
+  "canDelegate": null,
+  "condition": null,
+...
+  "type": "Microsoft.Authorization/roleAssignments"
+}
+```
+
+Once you do this, re-try the command that gave the error.
```

Comment:
Once you do this -> After you do this (once – don’t use to mean after; use only to mean less than twice.)

Reply:
Will do.